backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc

```diff
@@ -0,0 +1,10 @@
+[id="about-installing-oadp"]
+= About installing OADP
+include::modules/common-attributes.adoc[]
+:context: about-installing-oadp
+
+toc::[]
+
+You install the Openshift API for Data Protection (OADP) by installing the OADP Operator and then installing the Data Protection Application.
+
+You must have S3-compatible object storage for storing backups and a cloud provider that supports snapshots. You specify the backup location and snapshot location in the `DataProtectionApplication` custom resource manifest.
```

Comment:
> a cloud provider that supports snapshots
this is not quite correct. it also can use csi driver to do snapshot backup.

Reply:
Will update.